bin/cli.js

```diff
@@ -23,4 +25,11 @@ if (argv.all) {
     process.exit();
 }
 
-console.log(core.random());
+if (argv.say) {
+    console.log(cowsay.say({
+        text: wrap(core.random()),
+        f: path.join(__dirname, 'calvin.cow')
+    }));
+} else {
```

Comment:
Do the same as the above, putting `process.exit` in the if statement instead of an `else`. It's just a tiny thing to make it a little bit more readable.